app/helpers/jwt.js

```diff
@@ -0,0 +1,19 @@
+const jwt = require('jsonwebtoken');
+const config = require('../../config').common.api;
+const errors = require('../errors');
+
+exports.generateToken = payload => {
+  try {
+    return jwt.sign(payload, config.tokenSecret, { expiresIn: '1d' });
+  } catch (error) {
+    throw errors.apiError(error.message);
```

Comment:
Remember to add a log that shows the error and gives the custom error a more personalized message.

Reply:
This custom error is very generic, what do you think if you create a more specific one for this process?